### PR TITLE
Fix CanvasRenderer sprite.raycast bug

### LINKED ISSUE #14868: Raycaster no longer works with Sprites using CanvasRenderer
three.js  version more than 0.93, Raycaster Method no use to Sprite.

### PR DESCRIPTION
**Pull request by 06wj**
Fix #14868 

test: https://rawgit.com/06wj/three.js/dev/examples/canvas_interactive_particles.html

### CODE REVIEW

File: examples/js/renderers/CanvasRenderer.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ THREE.SpriteCanvasMaterial = function ( parameters ) {
 	THREE.Material.call( this );
 
 	this.type = 'SpriteCanvasMaterial';
-
+	this.rotation = 0;
```
Comment (Mugen87) [inline examples/js/renderers/CanvasRenderer.js]: I wonder if we should note that this property only ensures correct raycasting. It has currently no visual effect on sprites.

Comment (06wj) [reply examples/js/renderers/CanvasRenderer.js]: It used here.
https://github.com/06wj/three.js/blob/e64752d7d9e637351a1ee933072c6d82c60f4dab/examples/js/renderers/CanvasRenderer.js#L585

Comment (Mugen87) [reply examples/js/renderers/CanvasRenderer.js]: Uh, currently `_context.rotate()` is called with `undefined` as parameter. It's really good to add `rotation` to `SpriteCanvasMaterial` :sweat_smile:

**Review by Mugen87 — APPROVED** [review]

**Comment by WestLangley** [comment]
Since you added the `rotation` property, please update the docs. Compare with `SpriteMaterial.html`.

File: examples/js/renderers/CanvasRenderer.js (hunk 1 above)
Comment (mrdoob) [reply examples/js/renderers/CanvasRenderer.js]: Uh oh!

**Pull request merged**